patches/README.md

```diff
@@ -0,0 +1,13 @@
+# Patches apply in ascending order of file name
+
+Motherhen patches start with "0" and three more digits.  We believe these patches fix Mozilla build bustages for _any_ Motherhen-based project, at least for now.  We recommend starting your patches with four digits, where the leading digit is not "0".  (Alex would very much like to understand the ramifications of these patches better, and how we can build without them.  So, help wanted here!)
```

Comment:
The numbering is more for my sanity rather than any technical reason. When I used to work with patches in this format, I would get patches with names like [`browser-base-content-browser-sidebar-js.patch`](https://github.com/dothq/browser-desktop/blob/59195ff4eb58094ab31374ec8e23c7cb72d3e60e/patches/browser-base-content-browser-sidebar-js.patch). The ordering is to make sure you don't get a patch that appears earlier in the list that depends on a later patch.

In terms of the ramifications of each patch, here is what they do:
- `0001` & `0003`: Disable `services/settings`, which provides [Remote](https://firefox-source-docs.mozilla.org/services/settings/index.html) [Settings](https://remote-settings.readthedocs.io/en/latest/). Ideally, we would figure out a way to satisfy the remote settings module (e.g. A static, blank, remote settings instance that all motherhen points to by default + instructions on how to set up your own remote settings instance)
- `0002`: We would need to push a patch up to `mozilla-centeral` that correctly detects the existence of a gecko app in [`has_build`](https://searchfox.org/mozilla-central/source/python/mozbuild/mozbuild/base.py#1003) (the current check is `is_firefox or is_android or is_thunderbird` which is determined by the `MOZ_BUILD_APP` variable).

Reply:
I'll have to incorporate this into the README somehow.

Reply:
I'm going to largely cut & paste the above text into the README.  It's pretty good explanatory text.